js/course_settings_add_school_detail.js

```diff
@@ -35,7 +35,7 @@ function handleResponse(response) {
 
 function getAccountsPath(accounts) {
     return accounts.slice().reverse().map(function(account) {
-        return account.name;
+        return '<a href="'+window.location.origin+'/accounts/'+account.id+'">'+account.name+'</a>';
```

Comment:
As long as we never have to change base URLs here (we shouldn't?), I think we can skip the `window.location.origin` entirely and just use paths -- the browser will fill in the details (i.e. ["origin-relative URL"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_relative_urls)).

Reply:
Ah, good catch! Thanks!

Reply:
(Redeployed on Canvas QA)